repose-aggregator/commons/utilities/src/test/scala/org/openrepose/commons/utils/servlet/http/HttpServletRequestWrapperTest.scala

```diff
@@ -521,6 +523,7 @@ class HttpServletRequestWrapperTest extends FunSpec with BeforeAndAfter with Mat
       wrappedRequest.getPreferredHeader("ornament") shouldBe "santa"
     }
 
+    //todo: fix this test -- shouldn't the higher quality value be returned?
     it("should return the appropriate value when a higher quality is appended onto a line") {
```

Comment:
So i went that way at first, but headers read left to right which means the first semi-colon indicates the end of the value and the beginning of parameters.